AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/CascadeDeleteOperation.swift

```diff
@@ -221,6 +236,9 @@ public class CascadeDeleteOperation<M: Model>: AsynchronousOperation {
                 if let deleteResult = deleteResult {
                     switch deleteResult {
                     case .success:
+                        if modelName != "MutationEvent" {
+                            self.log.debug("[CascadeDelete.3] [Delete from Local Store] Local cascade delete of \(modelName) successful!")
```

Comment:
nit: why the brackets on "Delete From Local Store"? would suggest just "Local delete for \(modelName) successful". Not all models are cascade deleted either, only models with associations, so I would suggest dropping the "cascade delete" unless we have also determined that it is a cascade delete for this model